components/molecules/VideoCard/VideoCard.tsx

```diff
@@ -28,14 +26,14 @@ export interface VideoCardProps {
     creatorName?: string
     views?: number
     uploadedOn?: string
-    onPress?: () => void
+    onPress?: (event: any) => void
     onNext?: () => void
     showPlayButton?: boolean
     isVerified?: boolean
     style?: ViewStyle
-    video?: VideoModel
     playable?: boolean
     videoPreRollSource?: AVPlaybackSource
```

Comment:
This was being piped directly into the Video component, so two things:
1. We should remove this prop.
2. We should make sure the place where it was used still renders the preroll properly.